app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,168 @@
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+    If you train to teach physics, you could also be eligible receive a scholarship or bursary of up to £29,000.
+  </p>
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Spark curiosity in physics",
+      colour: "yellow",
+      image_1_path: "static/content/hero-images/0012.jpg",
+      image_1_alt: "Pupils listening to a science lesson.",
+      image_2_path: "static/content/hero-images/0007.jpg",
+      image_2_alt: "Teacher talking to a pupil in a classroom.",
+    ) do %>
+       <p>
+    As a physics teacher, you’ll be engaging students in a wide range of experiments that can spark their interest.
+  </p>
+
+  <p>
+    You can inspire your students to consider rewarding careers from tackling climate change to artificial intelligence.
+  </p>
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+      <p>
+      You'll teach the content of the national curriculum, with opportunities to develop your students’ ability to think like physicists and see physics as a solution to many challenges. The themes you are likely to cover include:
+      </p>
+      <p>
+          11 to 14 year olds (key stage 3):
+      </p>
+        <ul class="vertical-tags">
+          <li class="tag">Energy</li>
+          <li class="tag">Motion and forces</li>
+          <li class="tag">Electricity and electromagnetism</li>
+          <li class="tag">Waves and matter</li>
+        </ul>
+      <p>
+        14 to 16 year olds (key stage 4):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Energy, forces and wave motion</li>
+        <li class="tag">Electricity, magnetism and electromagnetism</li>
+        <li class="tag">The structure of matter</li>
+        <li class="tag">Atomic structure and space physics</li>
+      </ul>
+      <p>
+        16 to 18 year olds (key stage 5):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Fields, vectors and scalars</li>
+        <li class="tag">Mechanics and mechanical properties of matter</li>
+        <li class="tag">Electric circuits and waves</li>
+        <li class="tag">Matter, quantum and nuclear physics</li>
+      </ul>
+    <p>
+      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">Read the national curriculum for more information about physics lessons</a>.
+    </p>
+
+    <p>You can also search for past exam papers online to see the type of questions that have been covered.</p>
+
+    <p>
+      You may also need to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on the individual schools.
```

Comment:
```suggestion
      You may also get the opportunity to teach other science subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
```
can we specify 'science subjects' or could it be wider than that? also thought we could tweak the wording to make it seem like teaching other subjects can be a good thing!